api/core/model_runtime/model_providers/jina/jina.yaml

```diff
@@ -1,6 +1,6 @@
-provider: jina
+provider: Jina AI
```

Comment:
Please keep this name the same as the provider's path name, for this provider, it's jina.